src/az390.java

```diff
@@ -9115,22 +9116,20 @@ private void process_dcb_data(){
 				&& dc_field.charAt(dc_index) != ','){
 			    dc_index++;
 		}
-		if (dc_index >= dc_field.length()){
+		if (dc_index >= dc_field.length()){ 
 			log_error(65,"invalid binary dc data " + dc_field.substring(dc_data_start));
 			return;
 		}
-		dcb_len = dc_index - dcb_start;
+		dcb_bin = remove_blanks(dc_field.substring(dcb_start,dc_index)); // #233
```

Comment:
Don, I think performance would benefit if you first scan for any spaces occurring in the string, and only if  yes call the blanks-removal routine. Otherwise, just copy the input string value.
After all, the most prevalent case is the absence of spaces in the input string.

Reply:
Abe, I would agree if the strings were longer, but the average numberic field string is less than 8 characters so testing if bing for blank character would not save many if any cycles compared to just searching and removing blanks once for each numeric field.